database/sql/src/main/scala/cromwell/database/slick/tables/MetadataEntryComponent.scala

```diff
@@ -297,6 +298,18 @@ trait MetadataEntryComponent {
     } yield metadataEntry).size
   }
 
+  def metadataTableSizeInformation() = {
+    val query =
+      sql"""
+          |SELECT DATA_LENGTH, INDEX_LENGTH, DATA_FREE
+          |FROM information_schema.tables
+          |WHERE TABLE_NAME = 'METADATA_ENTRY'
+         """.stripMargin
+    query.as[InformationSchemaEntry](rconv = GetResult { r =>
+      InformationSchemaEntry(r.<<, r.<<, r.<<)
+    }).head
```

Comment:
I'll have to take your word on this... 😨

Reply:
I will update it to return `.headOption` instead 👍

Reply:
I meant all the `.<<`s, and so on! 😆 

I do like being careful in case it's not got any results though, so 👍  on this change anyway!